localisation/00_zuwc_united_localisation.csv

```diff
@@ -3095,6 +3095,14 @@ RULE_MISTS_OF_PANDARIA_ON;On;;Aktiviert;;;;;;;;;;;x
 RULE_MISTS_OF_PANDARIA_ON_DESC;The Mists of Pandaria are enabled, making Pandaria isolated until they part around the 613th year. ;;Die Nebel von Pandaria sind aktiviert, Interaktionen sind bis zum Jahr 613 blockiert.;;;;;;;;;;;x
 RULE_MISTS_OF_PANDARIA_OFF;Off;;Deaktiviert;;;;;;;;;;;x
 RULE_MISTS_OF_PANDARIA_OFF_DESC;The Mists of Pandaria are disabled, making Pandaria open to the world. ;;Die Nebel von Pandaria sind deaktiviert, Pandaria steht der Welt offen.;;;;;;;;;;;x
+emperor_of_pandaria;Blood of [This.GetFirstNameWithNick];;;;;;;;;;;;;x
+emperor_of_pandaria_desc;The next emperor of Pandaria after the legendary Shaohao, [This.GetFirstNameWithNick] restored the imperial traditions and united Pandaria once again.;;;;;;;;;;;;;x
+form_the_pandaria_empire;Restore the Pandaria Empire;;;;;;;;;;;;;x
```

Comment:
I've heard it called the Pandaren Empire, but not the Pandaria Empire. I feel like the Empire of Pandaria makes more sense in this context, as it is the empire ruling the land of Pandaria.
```suggestion
form_the_pandaria_empire;Restore the Empire of Pandaria;;;;;;;;;;;;;x
```

Reply:
Done in 51a4ac3c22c01f0b263199af3ba03e1d17d4b91c.